### ISSUE TITLE
Speaker classification problem

### ISSUE THREAD

**Issue by Miamoto** [s3prl/s3prl#291]
Hello! 

I am trying to run speaker utterance validation for librispeech just following your code. I am using as feature the mel one in the command. In the first 5k iterations the accuracy of the training is as shown: "best|libri_speaker/train-|step:5000|acc:0.005466052796691656". Will this really get to the 95.33 acc you mentioned in TERA paper starting with such a low acc number? Or am I not doing something correct that is not mentioned? 

This is the command I am using: "nohup python run_downstream.py -m train -u mel  -n result/downstream/speaker_mel -d speaker_linear_utter_libri"

Thanks a lot,
Carlos

**Comment by Miamoto** [comment]
Thanks for the answer! Well, do you have any idea how many steps to achieve 1% of acc? :) 

Thanks a lot,
Carlos

**Comment by andi611** [comment]
The acc growth is not linear, it will grow exponentially as training steps increase.

**Comment by Miamoto** [comment]
I have trained it for 300000 steps and the loss of the test set is 0.010358146391808987.

Should I train for more steps? I am just running the command above.

Thanks a lot,
Carlos

**Comment by andi611** [comment]
Hi sir!

@Miamoto Sorry for the delay,
I found my old training log, here is a snapshot:
![image](https://user-images.githubusercontent.com/25531495/158548897-1310e432-c90a-4804-97b7-6333e0cadf3c.png)
As shown in the last line, the **testing ACC is 95.33**.

Here is the full checkpoint and log files: [**google drive**](https://drive.google.com/drive/folders/1e5nAvFN6fh-ohPiZbFPPO7s5rEvGYkX8?usp=sharing)
Can you check that the config you are using is identical to mine?
I'm guessing you have `cmvn: True`,
however, I'm using this setting in my `config.yaml`:
```yaml
input:
  feat_type: mel # feat_type can be: wav, complx, linear, mel, mfcc, phase
  channel: 0
  log: True
  delta: 0
  cmvn: False
```
Let me know if you have any questions.

Andy

**Comment by leo19941227** [comment]
Feel free to re-open it!

**Comment by oakkas** [comment]
I am trying to test speaker_classification model and fine_tune it for a couple of speakers. I tried to understand the codebase but couldn't get it to run. Is there anyone who can guide me if there is a code/tutorial somewhere.
